shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -228,6 +233,7 @@ static void CommonInit(FlutterViewController* controller) {
   }
   controller->_additionalKeyResponders = [[NSMutableOrderedSet alloc] init];
   controller->_mouseTrackingMode = FlutterMouseTrackingModeInKeyWindow;
+  [controller setupNotificationCenterObservers];
```

Comment:
Do not call methods on an object during init. It's a common way to get subtle bugs over time. That's why this is `CommonInit` and not, say, `-doCommonInitialization`. If you want the code to be separate from this function for some reason, it should be in another, similar function.